pages/index.html

```diff
@@ -4,9 +4,48 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/styles/styles.css" rel="stylesheet">
+  <link href="/styles/index.css" rel="stylesheet">
+  <title>Kelly's Personal Portfolio</title>
 </head>
 <body>
+  <header>
+    <a id="logo" href="index.html">
+      <img src="/images/Logo.png" alt="Home" width="150">
+    </a>
+    <nav>
+      <ul>
+        <li><a href="about.html">About Me</a></li>
+        <li><a href="portfolio.html">My Projects</a></li>
+      </ul>
+      
+    </nav>
+  </header>
+  <section class="intro-section">
+    <div class="type-container">
+      <h1>Hello, there! I'm Kelly :)</h1>
+      <h3 id="typing">I am an aspiring software developer at Ada Developers Academy.</h3>
+    </div>
+    <div class="img-container">
+      <img src="/images/Me1.jpg" alt="My Picture" width="200">
+      <img src="/images/Me2.jpg" alt="My Picture" width="200">
+      <img src="/images/Me3.jpg" alt="My Picture" width="200">
+      
+    </div>
+    <h3>Browse around to learn more about me!</h3>
+    
+  </section>
+
+  <footer>
+    <div class="footer-container">
+      <div class="email">
```

Comment:
Since "footer-container" and "email" only occur once each in all your separate HTML files, it is okay to have them as IDs!